### PR TITLE
For #404, placed correct answer in (b) and commented reasoning for other false answers. Inserted correct limit of 1 in part (c).

### PR DESCRIPTION
**Pull request by jbunn3**

### CODE REVIEW

File: source/calculus/source/08-SQ/07.ptx [hunk 1]
```diff
@@ -103,19 +103,19 @@
...
+                    <li><m>\displaystyle \frac{(n+1)n^2}{n+2}</m>.</li> <!-- recip fail -->
                 </ol>                
             </p>
         </task>
         <task>
             <p>
                 Find <m>\displaystyle \lim_{n\to\infty} \left|\frac{a_{n+1}}{a_n}\right|.</m>
                 <ol marker="A." cols="2">
-                    <li><m>-\infty</m>.</li>
+                    <li><m>1</m>.</li><!-- ans -->
```
Comment (StevenClontz) [inline source/calculus/source/08-SQ/07.ptx:118]: Would be better to add an `<answer><p>A. <m>1</m></p></answer>` perhaps. Similarly, the comments above would be helpful as an `<answer/>` available in the instructor guide.

Comment (jbunn3) [reply source/calculus/source/08-SQ/07.ptx:118]: Ok, I haven't done anything yet with adding answers for the instructor guide. I will look into that.

Comment (StevenClontz) [reply source/calculus/source/08-SQ/07.ptx:118]: basically you want `<task><statement><p>stuff</p></statement><answer><p>answer</p></answer></task>`

Comment (jbunn3) [reply source/calculus/source/08-SQ/07.ptx]: @StevenClontz Thanks, I found it in the style guide too.

**Comment by jbunn3** [comment]
@cg2wilson I converted this to a draft, as I plan to work on adding the comments using `<answer> ` as well. I'll try to get that done next week. In the meantime, feel free to leave any feedback on the changes.

**Comment by cg2wilson** [comment]
things look good to me for when it gets marked as ready!

File: source/calculus/source/08-SQ/07.ptx [hunk 2]
```diff
@@ -95,33 +95,60 @@
...
+            </statement>   
+            <answer>
+                <p>
+                    D. <m> \displaystyle \frac{(n+1)^3}{(n+2)n^2}</m>.
+                </p>
+                <p>
+                    A. results from bad subscript usage.
+                    B. occurs from using the Freshman Dream Theorem: <m>(n+1)^2=n^2+1</m>.
+                    C. results from a bad cancellation.
+                    E. is the reciprocal of C.
```
Comment (StevenClontz) [inline source/calculus/source/08-SQ/07.ptx:130]:
```suggestion
                    <ul>
                        <li>A. results from bad subscript usage..</li>
                        <li>B. occurs from using the Freshman Dream Theorem: <m>(n+1)^2=n^2+1</m>..</li>
                        <li>C. results from a bad cancellation..</li>
                        <li>E. is the reciprocal of C.</li>
                    </ul> 
```

**Review by StevenClontz — APPROVED** [review]

**Pull request merged**